llpc/context/llpcPipelineContext.h

```diff
@@ -120,7 +120,9 @@ class PipelineContext {
   GfxIpVersion getGfxIpVersion() const { return m_gfxIp; }
 
   // Gets pipeline hash code
-  uint64_t getPiplineHashCode() const { return MetroHash::compact64(&m_pipelineHash); }
+  uint64_t getPipelineHashCode() const { return MetroHash::compact64(&m_pipelineHash); }
+  MetroHash::Hash getPipelineHashCodeWithoutCompact() const { return m_pipelineHash; }
+  void setPipelineHashCode(const MetroHash::Hash &hash) { m_pipelineHash = hash; }
```

Comment:
Can you leave a comment explaining what the difference between those two is and why we need both?

Reply:
done